### PR TITLE
wind speed widget

### LINKED ISSUE #60: wind speed html

### LINKED ISSUE #61: wind speed logic

### LINKED ISSUE #165: Wind Speed Tests

### PR DESCRIPTION
**Pull request by carsondrobe**

### CODE REVIEW

File: Site/bc_weather_station_dashboard/website/templates/weather.html [hunk 1]
```diff
@@ -152,9 +157,29 @@ <h2 class="card-text" id="snow-quality">N/A</h2>
                     <!-- Wind Speed Card -->
                     <div class="col-md-4">
                         <div class="card foreground">
+                            <div class="row justify-content-center align-items-center mb-0">
+                                <div class="col-auto">
+                                    <img src="{% static 'images/wind-icon.svg' %}" alt="Wind Speed Icon"
```
Comment (robertyac) [inline Site/bc_weather_station_dashboard/website/templates/weather.html]: Want to add this to the left of the title like the temp and humidity widget?

**Review by robertyac — CHANGES_REQUESTED** [review]
Nice looks really good.

Want to run a merge from main and I can add your tests to the CI automation?

File: Site/bc_weather_station_dashboard/website/templates/weather.html (hunk 1 above)
Comment (carsondrobe) [reply Site/bc_weather_station_dashboard/website/templates/weather.html]: Couldn't get that to work, maybe could use some help with it tmr in class

**Comment by carsondrobe** [comment]
Seems a map.js test is failing for windspeed now. Since i have individual tests for wind speed, how can i delete the old windspeed test from map.js

**Comment by carsondrobe** [comment]
Fixed Tests and added to Github CI/CD stuff

**Review by abijeet-dhillon — APPROVED** [review]
Nice, it fits well with the rest of our widgets!

**Review by robertyac — APPROVED** [review]
Awesome thanks for moving the icon abi

**Pull request merged**